sd-gui/src/parser.rs

```diff
@@ -24,6 +26,7 @@ impl UiLanguage {
         match self {
             Self::Chil => "chil",
             Self::Spartan => "spartan",
+            Self::LlvmIr => "llvm-ir",
```

Comment:
Same here, I think any user facing thing should read "LLVM" or "LLVM IR"

Reply:
Do you remember what these strings do anyway?